packages/nuejs.org/docs/tags.md

```diff
@@ -219,9 +219,9 @@ Options given as YAML:
 
   `poster` a URL for an image to be shown before the playback starts
 
-  `preload` a [hint to the browser][preload] on what to load prior playback
+  `preload` <span>a [hint to the browser][preload] on what to load prior playback</span>
```

Comment:
Would break because of flex on `.options` otherwise, as a new element is inserted, and flex breaks the text wrapping (most noticeable on small screens) and removes spaces.

Currently, for whatever reason, strips the underline from the link when wrapped in a span... :shrug: 

Previous comments:
- https://github.com/nuejs/nue/commit/752e680a73079859371a3732dd0b85df2dfe0186#r144440249
- https://github.com/nuejs/nue/commit/89aaa518494962c9bdf1f83e67f7c5a7d5b2eb4e#r144876994

Reply:
Without span:

![image](https://github.com/user-attachments/assets/dd019bb7-cb3b-4f7e-8562-e9a8fa69d84a)

With span:

![image](https://github.com/user-attachments/assets/ebe6b8bd-864e-4266-aa9d-defa7547d62a)

- missing underline on link
- does not break text flow